src/clientdlg.cpp

```diff
@@ -253,6 +253,8 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
 
     // set window title (with no clients connected -> "0")
     SetMyWindowTitle ( 0 );
+    // Set number of clients to track
```

Comment:
This comment should be a bit clearer. Why do we set clients to track? What do we track?

Reply:
Updated to `// track number of clients to detect joins/leaves for audio alerts`.